### DISCUSSION TITLE
Clearing Penumbra Cache

### DISCUSSION THREAD

**Discussion by Lunalicious** [xivdev/Penumbra#362] [Q&A] [answered]
Hello everyone,
I was wondering if it would be safe to completed remove the "Penumbra Cache" directory, as it has grown to be 290GB, which is quite a lot considering the Penumbra directory itself is just 7GB. And if it is safe, maybe a way in Penumbra itself would be a nice addition.

It also can be that I just did something horribly wrong and something is very off with my files...
Anyway, have a good day!

Luna

**Comment by NotNite** [comment, marked as answer]
There isn't a thing as the "Penumbra cache". You either mean one of two things:

- If you're referring to that one photo talking about moving between machines, the person who created it wrote the wrong text. That directory is the actual mod installations, and deleting it would delete all of your mods.
- The Mare cache, which can technically be deleted, but has its own caveats explained in the plugin's UI.

**Reply by Lunalicious** [reply]
Thank you very much, I was just blown away by my own stupidity
